### PR TITLE
fix: send all error information allong

### PR DESCRIPTION
**Pull request by edewit**
also see: https://github.com/axios/axios#handling-errors for more info.

fixes: #1100

### CODE REVIEW

File: packages/launcher-component/src/core/stuff.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function SpecialValue(props: { children: string }) {
 export function AlertError(props: { error: any }) {
   return (
     <Alert variant={AlertVariant.danger} title="Something weird happened:" aria-label="error-in-hub-n-spoke" style={{margin: '40px'}}>
-      {props.error.toString()}
+      {props.error.message}
```
Comment (ia3andy) [inline packages/launcher-component/src/core/stuff.tsx]: You have no guaranty that the error will have `props.error.message`. I would prefer is you the error sent by the client would implement `toString()` with the message.

File: packages/launcher-client/src/client/http.service.ts [hunk 2]
```diff
@@ -29,12 +29,12 @@ export class HttpService {
...
+    let message: string;
     if (reason && reason.message) {
-      errMsg = `An error occurred: ${reason.message}`;
+      message = `An error occurred: ${reason.message}`;
     } else {
-      errMsg = `Backend returned ${reason}`;
+      message = `Backend returned ${reason}`;
     }
-    throw new Error(errMsg);
+    return Promise.reject({...reason, message});
```
Comment (edewit) [inline packages/launcher-client/src/client/http.service.ts]: here there will always be a message

File: packages/launcher-component/src/core/stuff.tsx (hunk 1 above)
Comment (edewit) [reply packages/launcher-component/src/core/stuff.tsx]: see my other comment there will be a message plus the extra information as to why the error happend

File: packages/launcher-client/src/client/http.service.ts (hunk 2 above)
Comment (ia3andy) [reply packages/launcher-client/src/client/http.service.ts]: Yes but you have the component could be used with another error..

**Pull request merged**